x-pack/platform/packages/shared/kbn-cloud-security-posture/common/constants.ts

```diff
@@ -9,11 +9,17 @@ import { MisconfigurationEvaluationStatus } from './types/misconfigurations';
 
 export const KSPM_POLICY_TEMPLATE = 'kspm';
 export const CSPM_POLICY_TEMPLATE = 'cspm';
-export const CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN =
+
+export const CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_ALIAS =
+  'security_solution-cloud_security_posture.misconfiguration_latest';
+
+export const DEPRECATED_CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN =
   'logs-cloud_security_posture.findings_latest-default';
 export const CDR_LATEST_THIRD_PARTY_MISCONFIGURATIONS_INDEX_PATTERN =
   'security_solution-*.misconfiguration_latest';
-export const CDR_MISCONFIGURATIONS_INDEX_PATTERN = `${CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN},${CDR_LATEST_THIRD_PARTY_MISCONFIGURATIONS_INDEX_PATTERN}`;
+
+export const CDR_MISCONFIGURATIONS_INDEX_PATTERN = `${DEPRECATED_CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN},${CDR_LATEST_THIRD_PARTY_MISCONFIGURATIONS_INDEX_PATTERN}`;
```

Comment:
should we change the const name to reflect that it's deprecated? As in `DEPRICATED_CDR_MISCONFIGURATIONS_INDEX_PATTERN`